flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -125,7 +125,7 @@ run_test "State TTL RocksDb backend end-to-end test" "$END_TO_END_DIR/test-scrip
 
 run_test "Running Kerberized YARN on Docker test " "$END_TO_END_DIR/test-scripts/test_yarn_kerberos_docker.sh"
 
-run_test "SQL Client end-to-end test" "$END_TO_END_DIR/test-scripts/test_sql_client.sh"
+run_test "SQL Client end-to-end test" "$END_TO_END_DIR/test-scripts/test_sql_client.sh" "skip_check_exceptions"
```

Comment:
Why have you added this test to `run-pre-commit-tests.sh`? If there is some reason behind it, do it in separate commit and remove the runs from `run-nightly-tests.sh`.

Reply:
@pnowojski The reason for this is to validate the effect of the change in Travis, because if placed in `run-nightly-tests.sh', it will not be routinely executed. I was going to wait until the PR was approved to withdraw the change, but your suggestion may be more reasonable. I can split it into a separate commit.